### PR TITLE
Simplify PHP CS Fixer config

### PR DESCRIPTION
**Pull request by keradus**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | n/a
| License       | MIT
| Doc PR        | 

same rules, just written in easier way

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -6,18 +6,13 @@ if (!file_exists(__DIR__.'/src')) {
...
+        '@PHP71Migration' => true,
+        '@PHPUnit75Migration:risky' => true,        
         '@Symfony' => true,
         '@Symfony:risky' => true,
-        '@PHPUnit75Migration:risky' => true,
-        'php_unit_dedicate_assert' => ['target' => '5.6'],
-        'array_syntax' => ['syntax' => 'short'],
-        'fopen_flags' => false,
         'protected_to_private' => false,
         'native_constant_invocation' => true,
```
Comment (keradus) [inline .php_cs.dist:14]: enabling the rules as-is is causing the rule to use `strict => false` setting (ref https://cs.symfony.com/doc/rules/constant_notation/native_constant_invocation.html#strict )

when rule was introduced, on request of Sf, we set up `strict => true` for Symfony ruleset.
if you prefer `strict => false`, we can adjust the ruleset definition

what is your preference?

File: .php_cs.dist [hunk 2]
```diff
@@ -6,18 +6,13 @@ if (!file_exists(__DIR__.'/src')) {
...
-        'combine_nested_dirname' => true,
         'list_syntax' => ['syntax' => 'short'],
```
Comment (keradus) [inline .php_cs.dist:15]: FYI we gonna incorporate it in PHP migration ruleset https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5428

File: .php_cs.dist (hunk 1 above)
Comment (nicolas-grekas) [reply .php_cs.dist:14]: we now decided to always add the leading `\` to all constants

File: .php_cs.dist (hunk 2 above)
Comment (nicolas-grekas) [reply .php_cs.dist:15]: then let's remove this line too, and merge this PR after FriendsOfPHP/PHP-CS-Fixer#5428 is merged?

File: .php_cs.dist (hunk 1 above)
Comment (keradus) [reply .php_cs.dist:14]: ack, thanks for the confirmation

out of curiosity - what was the reason for this change?

File: .php_cs.dist (hunk 2 above)
Comment (keradus) [reply .php_cs.dist:15]: we need to extend integration tests to reflect this change. can't promise it's done in few days.
that's why i decided to have PR to cleanup what's currently possible, and do further cleanup in future, when changes on PHP CS Fixer are done and when fabbot is updated

File: .php_cs.dist (hunk 1 above)
Comment (nicolas-grekas) [reply .php_cs.dist:14]: measurable perf difference if I recall well

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @keradus.

**Pull request merged**